packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff

```diff
@@ -0,0 +1,19 @@
+- import { ThemeManager, ThemeProvider, defaultTheme } from "@kaizen/design-tokens"
++ import { KaizenProvider } from "@kaizen/components"
+
+- const themeManager = new ThemeManager(defaultTheme)
+
+...
+
+return(
+  <FrontendServices
+-     services={[...defaultPreset, ThemeProvider]}
++     services={[...defaultPreset, KaizenProvider]}
```

Comment:
Is `defaultPreset` meant to be `defaultTheme`?

Reply:
Nope, that's the FEF preset

Reply:
Oh. That's very confusing in this example. Maybe add a comment or put an example value for the variable?

Reply:
Hm, that value comes from a separate package so I don't think I want to elude people to thinking they should be managing that value. I'll add a description to mention this example is from the FE-Template, hopefully less confusion.